lessons/6_objects_intro/index.html

```diff
@@ -0,0 +1,332 @@
+document.querySelector(".book.name").innerText = name;
+document.querySelector(".book.price").innerText = price;
+document.querySelector(".book.pages").innerText = pageCount;
+          </code></pre>
+            </div>
+            <div class="fragment">
+            </div>
+
+        </section>
+
+        <section>
+            <h3>Time to add a second book!</h3>
+            <div class="fragment">
+                <div class="book-sample-img">
+                    <img src="zombie_raccoons.jpg">
+                    <img src="goats.jpg">
+                </div>
+                <br>
+                <div class="fragment">
+                    <pre><code data-trim>
+            let title1 = "Zombie Raccoons & Killer Bunnies";
+            let image1 = "zombie_raccoons.jpg";
+            let pageCount1 = 643;
+            let price1 = 20;
+
+            let title2 = "Raising goats for dummies";
+            let image2 = "goats.jpg";
+            let pageCount2 = 577;
+            let price2 = 15;
+          </code></pre>
+
+                </div>
+            </div>
+        </section>
+
+        <section class="">
+            <h3>Now it would be really great to put them all together!</h3>
+        </section>
+
+        <section>
+            <h3>OBJECTS TO THE RESCUE!!</h3>
+            <div class="fragment">
+            <pre><code data-trim>
+              let book = {
+                title: "Zombie Raccoons & Killer Bunnies",
+                image: "zombie_raccoons.jpg",
+                pageCount: 643,
+                price: 20
+              };
+            </code></pre>
+            </div>
+            <br/>
+            <div class="fragment">
+            <pre><code data-trim>
+document.querySelector(".book img").src = book.image;
+document.querySelector(".book.name").innerText = book.name;
+document.querySelector(".book.price").innerText = book.price;
+document.querySelector(".book.pages").innerText = book.pageCount;
+            </code></pre>
+            </div>
+        </section>
+
+        <section>
+            <h3>Objects in a nutshell</h3>
+            <ul>
+                <li class="fragment">Objects consist of keys and values</li>
+                <li class="fragment">The key is like the name of the variable</li>
+                <li class="fragment">The value is like a variable itself</li>
+            </ul>
+            <br/>
+            <div class="fragment">
+                            <pre><code>
+          let book = {
+            title: "Zombie Raccoons & Killer Bunnies",
+            image: "zombie_raccoons.jpg",
+            pageCount: 643,
+            price: 20
+          };
+        </code></pre>
+            </div>
+        </section>
+        <section>
+            <h2>What can we do with objects?</h2>
+        </section>
+        <section>
+            <h3>Get value of it's attributes</h3>
+            <pre><code data-trim>
+              let person = {
+                name: "Anna",
```

Comment:
I thinks it's better to stick with book example, because classwork is to work with person. Would be nice if student would need to name object variable properly as well. What do you think?

Reply:
will do

Reply:
done